docs/administration/configuration/hashicorp-consul.md

```diff
@@ -0,0 +1,49 @@
+# HashiCorp Consul (Enterprise)
+
+Rundeck Enterprise users have the option to use HashiCorp Consul as a method to edit Rundeck's configuration properties. By doing so, it allows users to manage the configuration for all of their services in one place. In terms of Rundeck, this feature is very beneficial because it allows users to edit configurations across all of their Rundeck instances. So, instead of having to change a configuration for each instance, you can change it once in Hashicorp and it will apply to all instances.  
+
+## Getting Started 
+
+To begin, we need to make sure that we have both Rundeck and HashiCorp Consul installed. The easiest way to get HashiCorp up and running is using their docker image. The docker image is named "consul." Consul can be found at localhost:8500, but this can also be changed from within Consul. 
+We also need to make sure that Rundeck knows to use Consul for configuration. There are a couple of different ways to do that:
+
+### RPM, Debian and Docker Users
+
+For RPM, Debian and Docker users, the best way to specify that Rundeck should use Consul, is by defining the environment variable to be true. To do so, we need to add the following to the `/etc/rundeck/profile` file:
+```
+RUNDECK_CONSUL_ENABLED=true
+```
+
+### War Users
+
+For customers who use installed Rundeck using the .war, there are a couple different options.
+1. You could define the environment variable just like RPM, Debian and Docker users by adding the following to the `/etc/rundeck/profile` file:
+2. You could pass the property as a flag when starting Rundeck. An example of that would look like the following:
+```
+java -Xmx4g -Drundeck.consul.enabled=true -jar rundeck-{{{rundeckVersionFull}}}.war
+```
+
+### Updating the Server Name
+
+Now that you have specified to use Consul, we can shut down Rundeck and look at the newly created `bootstap.yml` file. In there, it will specify the host of Consul. This should be updated to the IP address or domain name you are using. 
+
+## Services
```

Comment:
Somehow this section and the next section need to be marked to show they are found in the Consul user interface. Otherwise the user might think these are found in Rundeck somewhere. Perhaps show a screenshot of the Consul home page and the two tabs you can click on to go to these specific areas.